app/konnect/dev-portal/register-and-create-app.md

```diff
@@ -0,0 +1,79 @@
+
+**Developer platform:** The developer platform allows developers like you to locate, access, and consume APIs. Using the Dev Portal, you can browse and search API documentation, test API endpoints, and manage your own credentials. 
+{% endcapture %}
+
+{{ konnect_concepts | markdownify }}
+
+</details>
+
+## Register and create an application
+
+All developers must register through the {{site.konnect_short_name}} Dev Portal. A {{site.konnect_short_name}} admin can provide you with the correct registration URL.
+
+1. **Register or sign in to the developer platform**
+    
+    Navigate to Dev Portal and do one of the following:
+    * Register for access by navigating to the Dev Portal and creating an account.
+    * Sign in to the Dev Portal with single sign-on (SSO).
+
+    Once you're approved for access, you can create an application in Dev Portal.
+
+1. **Create an application**
+
+    Click **My Apps**. From the **My Apps** page, click **New App** and fill out the application information. 
+    
+    {:.note}
+    > **Note:** The Reference ID must be unique. If your organization is using the [OIDC](/konnect/dev-portal/applications/enable-app-reg#oidc-flow) flow for application registration, enter the ID of your third-party OAuth2 claim.
+
+1. **Add products to your application**
+
+    Click **Catalog**, click the product version you want to create an application for, and then click **Register**. In the registration pop-up, select the app you just created and click **Request Access**. Repeat this for as many product versions as you want to add to one application.
+
+    Each application can only work with one auth strategy. Dev Portal can contain products with different auth strategies, so keep this in mind when adding more than one product to your application.
+    
+    {:.note}
+    > **Note:** The Reference ID must be unique. If your organization is using the [OIDC](/konnect/dev-portal/applications/enable-app-reg#oidc-flow) flow for application registration, enter the ID of your third-party OAuth2 claim.
+
+1. **Create API credentials for your application**
+
+    Every application must have credentials so that you can access your app. An application can have multiple credentials. You can generate credentials using one of the following methods:
+    * **Generate credentials**: A credential, or API key, generated in the {{site.konnect_short_name}} Dev Portal is a 32-character string associated with an application. 
+        1. To generate an API key from the Dev Portal, navigate to the **My Apps** page and click the application that you want to generate a credential for. 
+
+        1. In the **Authentication** pane, click **Generate Credential**.
+        
+    * **Manually create OIDC credentials**: To use ODIC without DCR, you must manually create the application in an IdP and use the reference ID (listed on your application in Dev Portal) when creating the application.
+
```

Comment:
```suggestion
        1. To generate an API key from the Dev Portal, navigate to the **My Apps** page and click the application that you want to generate a credential for. 

        1. In the **Authentication** pane, click **Generate Credential**.
        
    * **Manually create OIDC credentials**: To use ODIC without DCR, you must manually create the application in an IdP and use the reference ID (listed on your application in Dev Portal) when creating the application.

```

Would it make sense to link to https://deploy-preview-7450--kongdocs.netlify.app/konnect/dev-portal/applications/dynamic-client-registration/

Or any of the OIDC docs for the OIDC step?